### PR TITLE
Feature/rm 8139 change text properties web tree view boc tree view web string

### PR DESCRIPTION
**Pull request by a-ctor**
https://re-motion.atlassian.net/browse/RM-8139

### CODE REVIEW

File: Remotion/Web/Core/WebStringConverter.cs [hunk 1]
```diff
@@ -100,7 +100,7 @@ private object ConvertToString (in WebString input)
 
       if (input.Type == WebStringType.PlainText)
       {
-        if (rawString.StartsWith (c_htmlPrefix))
+        if (rawString.StartsWith (c_htmlPrefix) || rawString.StartsWith (c_textPrefix))
```
Comment (MichaelKetting) [inline Remotion/Web/Core/WebStringConverter.cs:103]: Why this change?

File: Remotion/Web/UnitTests/Core/WebStringConverterTest.cs [hunk 2]
```diff
@@ -66,6 +66,7 @@ public void ConvertToString (WebString? input, string expectedOutput)
         new object[] { WebString.CreateFromText (string.Empty), string.Empty },
         new object[] { WebString.CreateFromText ("test"), "test" },
         new object[] { WebString.CreateFromText ("(html)test"), "(text)(html)test" },
+        new object[] { WebString.CreateFromText ("(text)test"), "(text)(text)test" },
```
Comment (MichaelKetting) [inline Remotion/Web/UnitTests/Core/WebStringConverterTest.cs:69]: See https://github.com/re-motion/Framework/pull/261/files#diff-fe47fd88d0742bdd44b6142569bf1c5762f84c8178e53b12c080c2b1d7d65aaaL103-R103

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

File: Remotion/Web/Core/WebStringConverter.cs (hunk 1 above)
Comment (a-ctor) [reply Remotion/Web/Core/WebStringConverter.cs:103]: Without it the conversion is not correct in the special case that the text starts with `(text)`. For example:
 - we have a WebString with text `(text)asd` and a type plaintext
 - we convert it to a `string` using `ConvertTo` which as of now will return `(text)asd` instead of `(text)(text)asd`
 - we convert it back to a `WebString` using `ConvertFrom` but this will remove the `(text)` prefix leaving us with a web string with text `asd` type plaintext

**Review by MichaelKetting — APPROVED** [review]
I guess this means we're done?

**Comment by a-ctor** [comment]
I guess so
